ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobMetadata.java

```diff
@@ -52,7 +54,8 @@ public CloudBlobMetadata() {
    * @param expirationTime The blob expiration time.
    * @param size The blob size.
    */
-  public CloudBlobMetadata(BlobId blobId, long creationTime, long expirationTime, long size) {
+  public CloudBlobMetadata(BlobId blobId, long creationTime, long expirationTime, long size,
+      boolean usesCloudEncryption) {
```

Comment:
Don't you need to say: this. usesCloudEncryption = usesCloudEncryption ?

Reply:
added